primitives/tracing/src/logging.rs

```diff
@@ -33,7 +33,7 @@ use tracing_subscriber::{
 /// Span name used for the logging prefix. See macro `sc_cli::prefix_logs_with!`
 pub const PREFIX_LOG_SPAN: &str = "substrate-log-prefix";
 
-pub(crate) struct EventFormat<T = SystemTime> {
+pub struct EventFormat<T = SystemTime> {
```

Comment:
Why does this file is now duplicated?

Reply:
While working out some things I needed access to some of the types here so was considering a refactor to move this to the sc_tracing crate - it needs resolving

Reply:
@mattrutherford I am confused. Do you mean moving this to the `sp_tracing` or `sc_tracing` crate? In #7328 @cecton added this stuff to `cli/src/logging.rs`.

Reply:
I meant to `sc_tracing` because it's client/tracing related code - however, the way we're implementing it now means it's not required to be there - so a refactor would be optional and probably out of scope for this PR.

Reply:
Actually I should probably move those in something common (sc-service I guess) that I can share with substrate-browser-utils. I will investigate that on my PR #7463 because it's blocking